CHANGES_NEXT_RELEASE

```diff
@@ -4,4 +4,4 @@ Change HA model for execution of actions
 Fix error if parameter "type" exist in Update entity action (#30)
 Added 'twitter' action
 Changed 'update' action, any entity
-
+Do not allow 'id' and 'type' as attribute names
```

Comment:
The corresponding .md file should explain that id and type are forbidden as attribute names.